docs/source/getting_started/install.rst

```diff
@@ -25,13 +31,25 @@ with this command:
    pip install ansys-dpf-core<0.3.0
 
 
-Install using a wheel file
---------------------------
+Install without internet
+------------------------
+
+If you are unable to install PyDPF-Core on the host machine using ``pip`` due to
+network isolation, download the wheelhouse corresponding to your platform and Python interpreter version
+for the latest release of PyDPF-Core from the assets section of the `latest PyDPF-Core release on GitHub <https://github.com/pyansys/pydpf-core/releases/latest>`_. 
+
+The wheelhouse is a ZIP file containing Python wheels for all the packages PyDPF-Core requires to run.
+To install PyDPF-Core using the downloaded wheelhouse, unzip the wheelhouse to a local directory,
+then use the following command from within this local directory:
+.. code::
+
+   pip install --no-index --find-links=. ansys-dpf-core`
+
+Beware that PyDPF-Core wheelhouses do not include the optional plotting dependencies.
+To allow for plotting capabilities, also download the wheels corresponding to your platform and Python interpreter version
+for `PyVista <https://pypi.org/project/pyvista/#files>`_ and 
```

Comment:
Maybe we could mention the versions for PyVista and Matplotlib (the url points to the latest)

Reply:
@rafacanton the thing is these are going to change all the time. We are supposed to support the latest versions of each.
The current limitation on PyVista is actually only for the documentation generation pipeline on GitHub since there is an issue I have not been able to reproduce locally.

Reply:
@PProfizi OK, if we are always going to support the last one, then it LGTM!